packages/orbit-tailwind-preset/src/presets/components/index.ts

```diff
@@ -0,0 +1,158 @@
+import type { Config } from "tailwindcss";
+import { defaultTokens } from "@kiwicom/orbit-design-tokens";
+import plugin from "tailwindcss/plugin";
+
+import {
+  kebabCase,
+  getComponentLevelToken,
+  ExportedComponentLevelTokens,
+} from "../foundation/helpers";
+
+const COLORS: Partial<ExportedComponentLevelTokens>[] = [
+  "button",
+  "buttonLink",
+  "socialButton",
+  "alert",
+  "icon",
+  "formElement",
+  "badge",
+  "tag",
+  "textLink",
+  "text",
+  "heading",
+];
+
+interface Options {
+  /** default: true e.g. does not include default normalize styles */
+  disablePreflight?: boolean;
+  /** default: [] */
+  content?: Config["content"];
+}
+
+const cfg = (options?: Options): Config => {
+  const { disablePreflight = true } = options || {};
+
+  return {
+    content: ["auto"],
+    corePlugins: {
+      preflight: disablePreflight ? false : undefined,
+    },
+    theme: {
```

Comment:
I think the content of this object needs to be wrapped in `extends: { … }`. Otherwise it will replace the original tailwind classes, which is something we don't want to do